src/main/resources/interface-specification.yml

```diff
@@ -1128,18 +1128,21 @@ components:
         code:
           type: string
           description: source identifier
-          pattern: '[a-z0-9 \-]{,48}'
-          maxLength: 48
+          pattern: '^[a-zA-Z0-9\-_]+$'
```

Comment:
I see that in other regex the `_` is escaped, but not here.
Is this intentional?

Reply:
☝️

Reply:
Removing the escape in the other patterns is not necessary.

Reply:
why?

Reply:
Is not necessary to escape "_".
Am I missing something?

Reply:
Hmm, tests with regex101.com using the ecmascript parser confirmed what you're saying